qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/presenter.py

```diff
@@ -242,8 +228,22 @@ def set_errorbars_tab_enabled(self):
         enable_errorbars = curve_has_errors(self.get_selected_curve())
         self.view.set_errorbars_tab_enabled(enable_errorbars)
 
-    def update_view(self):
+    def on_axes_index_changed(self):
+        self.update_view(update_axes=False)
```

Comment:
I'm a bit confused why in `on_axes_index_changed` we specifically don't update the axis but just the curves, and why `on_curves_index_changes` updates nothing.


Could we put a comment so that people understand why this is False, and why it isn't a bug?